backend/app/domain/game.py

```diff
@@ -2,7 +2,7 @@
 from typing import List, Optional
 from uuid import uuid4
 
-from app.dto import PlayerDto
+from app.dto import PlayerDto, Investigator
```

Comment:
test add comment when pending from github, it is visible ONLY to me ?

Reply:
tes t again

Reply:
now, leave a comment then click fxxking `start a review` , this comment should be pending and nobody can see it

Reply:
pending again 2

Reply:
pending again 3

Reply:
github forces you in pending mode, so all the comments from here cannot be visible to other people.

Reply:
until you click the button `finish your review` at top-right corner, 
then all the pending comments above will be visible to everyone.

Reply:
I dunno when should you postpone some comments in GIthub PR review, 
what is the good use cases for that ?